code/game/objects/items/devices/radio/radio.dm

```diff
@@ -197,14 +203,26 @@
 			var/obj/item/clothing/gloves/radio/G = mute.get_item_by_slot(ITEM_SLOT_GLOVES)
 			if(!istype(G))
 				return FALSE
-			if(length(empty_indexes) == 1)
-				message = stars(message)
 			if(length(empty_indexes) == 0) //Due to the requirement of gloves, the arm check for normal speech would be redundant here.
 				return FALSE
 			if(mute.handcuffed)//Would be weird if they couldn't sign but their words still went over the radio
 				return FALSE
 			if(HAS_TRAIT(mute, TRAIT_HANDS_BLOCKED) || HAS_TRAIT(mute, TRAIT_EMOTEMUTE))
 				return FALSE
+			if(length(empty_indexes) == 1)
+				message = stars(message)
+
+//Start of Mojave Sun edit
+	if (radio_broadcast >= 0)
+		if (radio_broadcast == 0)
+			return FALSE
+		if (radio_broadcast == 1)
```

Comment:
Rather than having radio_broadcast define tiers (0-3), that represent different values (1-14):

1.  Replace your magic numbers (1-14) with defines - i.e `#define RADIOSTATIC_LIGHT 1`
2. set your radio_broadcast to your defines instead of tiers - i.e `radio_broadcast = STATIC_HEAVY`
3. feed your radio_broadcast straight into stars() i.e
```
if(radio_broadcast)
    message = stars(message, radio_broadcast)
```